changes.go

```diff
@@ -778,11 +778,14 @@ func (s *ChangesService) DeleteTopic(changeID string) (*Response, error) {
 	return s.client.DeleteRequest(u, nil)
 }
 
-// DeleteDraftChange deletes a draft change.
+// DeleteChange deletes a new or abandoned change
 //
-// Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#delete-draft-change
-func (s *ChangesService) DeleteDraftChange(changeID string) (*Response, error) {
-	u := fmt.Sprintf("changes/%s", changeID)
+// New or abandoned changes can be deleted by their owner if the user is granted the Delete Own Changes
+// permission, otherwise only by administrators.
+//
+// Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#delete-change
+func (s *ChangesService) DeleteChange(changeID string) (*Response, error) {
+	u := fmt.Sprintf("changes/%s/", changeID)
```

Comment:
Is there a reason for the trailing `/`?
According to the doc example, a slash is not needed.
In similar calls, we don't add on either.

```suggestion
	u := fmt.Sprintf("changes/%s", changeID)
```